packages/ragbits-core/src/ragbits/core/prompt/promptfoo.py

```diff
@@ -21,6 +21,9 @@ def generate_configs(
         root_path: The root path to search for Prompt objects. Defaults to the directory where the script is run.
         target_path: The path to save the promptfoo configuration files. Defaults to "promptfooconfigs".
     """
+    if not root_path:
+        root_path = Path.cwd()
```

Comment:
Can we be consistent with this?  Now we have three ways of doing it, above we use `root_path = root_path or Path.cwd()`, below we have a seprate func for it (`_get_current_dir`). Let's choose one and stick to it